packages/product-feed-google/src/Model/FeedItem/GoogleFeedItemFactory.php

```diff
@@ -62,33 +67,36 @@ public function create(Product $product, DomainConfig $domainConfig): GoogleFeed
      */
     protected function getBrandName(Product $product): ?string
     {
-        $brand = $product->getBrand();
-
-        return $brand !== null ? $brand->getName() : null;
+        return $product->getBrand()?->getName();
     }
 
     /**
      * @param \Shopsys\FrameworkBundle\Model\Product\Product $product
      * @param \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig $domainConfig
      * @return \Shopsys\FrameworkBundle\Model\Pricing\PriceInterface
      */
-    protected function getPrice(Product $product, DomainConfig $domainConfig): PriceInterface
+    protected function getBasicPrice(Product $product, DomainConfig $domainConfig): PriceInterface
     {
-        return $this->productPriceCalculationForCustomerUser->calculateBasicPriceForCustomerUserAndDomainId(
+        $defaultPricingGroup = $this->pricingGroupSettingFacade->getDefaultPricingGroupByDomainId($domainConfig->getId());
+
+        return $this->productPriceCalculation->calculatePrice(
             $product,
```

Comment:
why do not we use productPriceCalculationForCustomerUser->calculatePricesForCustomerUserAndDomainId here and extract the basic/special price from that?

Reply:
Ah, ok, thanks for the clarification. I did not notice that